__tests__/app.test.js

```diff
@@ -232,3 +232,24 @@ describe('DELETE /api/comments/:comment_id', () => {
 
 })
 
+describe.only('GET /api/users', () => {
```

Comment:
You should remove .only before pushing so you and we know that all your other tests are passing!